openassessment/staffgrader/errors/submission_lock.py

```diff
@@ -7,3 +7,7 @@ class SubmissionLockContestedError(Exception):
     """
     Error indicating trying to modify a lock that the user does not have access to modify.
     """
+    error_code = 'ERR_LOCK_CONTESTED'
+
+    def __str__(self):
```

Comment:
I would maybe not have this be the __str__ for this exception? Maybe introduce a new method like "get_error_code()" or something? Any thoughts on that?

Reply:
I had played with a few option. I did this just so we can do the same `str(ex)` for any exception type but it is a bit hacky.